README.md

```diff
@@ -102,6 +103,22 @@ with slurmdbdconfig.edit("/etc/slurm/slurmdbd.conf") as config:
     del config.auth_alt_parameters
 ```
 
+##### `cgroupconfig`
+
+This module provides an API for editing _cgroup.conf_ files, and create new _cgroup.conf_
+files if they do not exist. Here's some operations you can perform on the _cgroup.conf_
+file using this editor:
```

Comment:
Hmm, I'm reading these two side-to-side and I think it would be nicer if they were grammatically consistent. Something like "This module provides an API for editing <TYPE> files, and creating new <TYPE> files if they do not exist. Here's some operations you can perform on <TYPE> files using this editor:".

Reply:
Sure, I can revise this quickly. Also, with this change, the less likely in future that we'll forget to update the configuration file name when we add new sections for even more editors.